README.md

```diff
@@ -179,6 +179,10 @@ a suggestion is attempted to be highlighted either by the keyboard or mouse. It
 scope, access to an object called `suggestion` which is the current suggestion that is being
 interacted with. A non-selectable suggestion cannot be clicked on, hovered over, or interacted with
 via the keyboard.
+ - `inputHint({query}) {Promise}`: An expression that hints at a suggestion by resolving a
```

Comment:
It does not leave out that possibility. The app-maker can totally make two AJAX requests in one `sourceFn` -- he/she just has to combine the promises.